medusa/__main__.py

```diff
@@ -1247,6 +1258,27 @@ def path_leaf(path):
                 folder_path = os.path.join(cache_folder, name)
                 helpers.remove_folder(folder_path)
 
+    def migrate_python_version(self, new_version):
+        """
+        Perform some cleanup in case we switch from python version.
+
+        It's possible to switch from python version 2 to 3 or visa versa.
+        In that case we might wanna run some sanitary actions, to make sure everything keeps working.
+        :param new_version: currently started python version
+        :type new_version: int
+        """
+        # TODO: Leaving this here as a template for when we upgrade merge the python3 changes.
+        if new_version == 3:
+            pass
+        else:
+            pass
+
+        app.PYTHON_VERSION = new_version
```

Comment:
If this is only going to save the major version, I'd rather have this named `MAJOR_PYTHON_VERSION`. Otherwise I'd like to have the full version saved (even better actually).

Reply:
Okay, i can store as a list